### ISSUE TITLE
Onboarding tells you to download the Desktop app even though you're already using it

### ISSUE THREAD

**Issue by t3chguy** [vector-im/element-web#23112]

**Comment by gsouquet** [comment]
@niquewoodhouse We've received this piece of feedback. Are there actions you think we can take to improve the current situation?

**Comment by gsouquet** [comment]
@justjanne I'm not sure we would consider this task as completed if running in Electron.
My hunch is that the goal of this step is to have two sessions enabled. This is particularly important for recovery and ensuring you won't end up loosing all your history if you haven't backed up properly your decryption keys

**Comment by justjanne** [comment]
I agree, but in case we actually do decide that that's the route we want to choose, I want to have a PR ready with checks completed in time for the RC.

**Comment by niquewoodhouse** [comment]
> @justjanne I'm not sure we would consider this task as completed if running in Electron.
> My hunch is that the goal of this step is to have two sessions enabled. This is particularly important for recovery and ensuring you won't end up loosing all your history if you haven't backed up properly your decryption keys

I think you're right too, but maybe for other reasons.  The objective is increase retention and we have a hypothesis that mobile sessions will increase retention.  Maybe our copy could be clearer

'Download Element' 
'Try Element on your computer and phone' or something that explicit might help

**Comment by niquewoodhouse** [comment]
I realise this isn't necessarily a thing we intend to work on right now, given the ongoing threads efforts, but just mentioning @justjanne for when this next comes up.

Design proposal is to change the copy:

'Download Element'
'Try Element on your computer and phone, so you don't miss a thing'

And try to have it so a mobile session and a desktop session are the requirements for a tick.

**Comment by t3chguy** [comment]
> And try to have it so a mobile session and a desktop session are the requirements for a tick.

We aren't able to tell the type of devices your other sessions are, only their name, which varies per language & platform, and is ultimately user-configurable. Checking based on the name would cause the tick to disappear if you change the name of a device
